tests/functional/rbac/test_reassignment.py

```diff
@@ -149,6 +154,137 @@ def grant_role(self, client: ADCMClient, user: User, role: RbacRoles, *objects:
+            test_group = clients.admin.group_create("Test_group", user=[{"id": user.id}])
+        with allure.step('Create a test policy with user group'):
+            test_policy = clients.admin.policy_create(
+                name='Test_policy',
+                role=clients.admin.role(name=RbacRoles.ClusterAdministrator.value),
+                user=[],
+                group=[test_group],
+                objects=[admin_cluster],
+            )
+        user_cluster, user_service, user_component = as_user_objects(
+            clients.user, admin_cluster, admin_service, admin_component
+        )
+        is_allowed(user_cluster, BusinessRoles.EditClusterConfigurations)
+        is_allowed(user_service, BusinessRoles.EditServiceConfigurations)
+        is_allowed(user_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Change test group: delete user'):
+            test_group.update(user=[])
+        is_denied_to_user(admin_cluster, BusinessRoles.EditClusterConfigurations)
+        is_denied_to_user(admin_service, BusinessRoles.EditServiceConfigurations)
+        is_denied_to_user(admin_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Change test policy: add user'):
+            test_policy.update(user=[{"id": user.id}])
+        is_allowed(user_cluster, BusinessRoles.EditClusterConfigurations)
+        is_allowed(user_service, BusinessRoles.EditServiceConfigurations)
+        is_allowed(user_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Change test policy: delete user'):
+            test_policy.update(user=[])
+        is_denied_to_user(admin_cluster, BusinessRoles.EditClusterConfigurations)
+        is_denied_to_user(admin_service, BusinessRoles.EditServiceConfigurations)
+        is_denied_to_user(admin_component, BusinessRoles.EditComponentConfigurations)
+
+    def test_add_remove_cluster_from_policy(self, clients, is_denied_to_user, prepare_objects, user):
+        """Test that policies are applied after cluster add/remove to the policy"""
+
+        admin_cluster, *_ = prepare_objects
+        role_to_check = BusinessRoles.EditClusterConfigurations
+
+        with allure.step("Create two clusters"):
+            bundle = upload_bundle(clients.admin, 'cluster')
+            first_cluster = bundle.cluster_create(name='Test Cluster 1')
+            second_cluster = bundle.cluster_create(name='Test Cluster 2')
+
+        is_denied_to_user(admin_cluster, role_to_check)
+        is_denied_to_user(first_cluster, role_to_check)
+        is_denied_to_user(second_cluster, role_to_check)
+
+        with allure.step('Create a test policy with user and first cluster'):
+            test_policy = clients.admin.policy_create(
+                name='Test_policy',
+                role=clients.admin.role(name=RbacRoles.ClusterAdministrator.value),
+                user=[user],
+                objects=[first_cluster],
+            )
```

Comment:
I guess soon it'll be time to finally make a wrapping for `RbacRoles` based policy creation :D